src/hamcrest/__init__.py

```diff
@@ -1,7 +1,13 @@
 from hamcrest.core import *
+from hamcrest.core import __all__ as _core_vars
 from hamcrest.library import *
+from hamcrest.library import __all__ as _library_vars
 
 __version__ = "2.0.3.dev0"
 __author__ = "Chris Rose"
 __copyright__ = "Copyright 2020 hamcrest.org"
 __license__ = "BSD, see License.txt"
+
+__all__ = []
+__all__.extend(_core_vars)
```

Comment:
If we're going down this route, I'd rather see us use this syntax instead: 
```python
from . import core
__all__.extend(core.__all__)
```

Reply:
Fixed.